### PR TITLE
added generated files

### PR DESCRIPTION
**Pull request by jamesfan1**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Nginx/Nginx.Autorest/README.md [hunk 1]
```diff
@@ -29,18 +26,18 @@ For information on how to develop for `Az.Nginx`, see [how-to.md](how-to.md).
...
   - $(this-folder)/../../readme.azure.noprofile.md
 input-file:
 # You need to specify your swagger files here.
-  - $(repo)/specification/nginx/resource-manager/NGINX.NGINXPLUS/stable/2023-04-01/swagger.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/nginx/resource-manager/NGINX.NGINXPLUS/preview/2024-01-01-preview/swagger.json
 
 root-module-name: $(prefix).Nginx
 title: Nginx
-module-version: 0.1.0
+module-version: 0.1.1
```
Comment (VeryEarly) [inline src/Nginx/Nginx.Autorest/README.md:40]: please leave this version as it was

File: src/Nginx/Nginx.Autorest/examples/Invoke-AzNginxAnalysisConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
```
Comment (VeryEarly) [inline src/Nginx/Nginx.Autorest/examples/Invoke-AzNginxAnalysisConfiguration.md:1]: please fill out this file and run build-module.ps1 to update the doc

**Comment by VeryEarly** [comment]
please re-record tests after API upgrade

File: src/Nginx/Nginx.Autorest/test/utils.ps1 [hunk 3]
```diff
@@ -60,13 +61,80 @@ function setupEnv() {
...
+    $nginxDeployment.Name | Should -Be $env.nginxDeployment1
+
+    # assigning role
+    $keyVaultId = $keyVault.ResourceId
+    $roleDefinition = Get-AzRoleDefinition -Name "Key Vault Administrator"
+    $roleAssignment = New-AzRoleAssignment -ObjectId $nginxDeployment.IdentityPrincipalId  -RoleDefinitionId $roleDefinition.Id -Scope $keyVaultId
+    
 }
 function cleanupEnv() {
     # Clean resources you create for testing
```
Comment (VeryEarly) [inline src/Nginx/Nginx.Autorest/test/utils.ps1:139]: you might want to clean up resources created in setupEnv() here

Comment (jamesfan1) [reply src/Nginx/Nginx.Autorest/test/utils.ps1:139]: fixed, thank you

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**